WordPress/wporg-mu-plugins
# Header: Output just the variables generated by `theme.json` from the News site

Updates the classic header to output the custom properties set by the News theme. This removes the network option method, in favor of directly calling `wp_get_global_stylesheet` on News.

It likely still needs to be cached, but I wanted a gut-check on the approach first :)

There's a workaround for the `--wp--style--block-gap` property, since it's not considered part of the `variables` set, so instead I'm setting it directly. If that changes for some reason, it'll need to be changed here too.

Fixes #66 

To test:

- Needs to be tested on a sandbox
- Visit a classic theme site, ex: [wordpress.org/showcase/](wordpress.org/showcase/)
- In this site, the `h3`s should have the correct (sans-serif) font
- View source, find the style tag `id=global-styles-for-classic-themes`, it should only have custom properties on `body`, no other styles
